js/stringProcessing/stripHTMLTags.js

```diff
@@ -3,13 +3,30 @@
 var stripSpaces = require( "../stringProcessing/stripSpaces.js" );
 
 /**
+ * Strip incomplete tags within a text. Strips an endtag at the beginning of a string and the start tag at the end of a
+ * start of a string.
+ * @param {String} text The text to strip the HTML-tags from at the begin and end.
+ * @returns {String} The text without HTML-tags at the begin and end.
+ */
+var stripTagParts = function( text ) {
```

Comment:
I'd rename this to better encompass what it's stripping. Consider naming it something like `stripIncompleteTags` instead of the current name.